config.ini

```diff
@@ -19,7 +60,7 @@ Connection: keep-alive
 Referer: http://myanimelist.net/forum/?clubid=379
 
 [jikanpy]
-request-interval : 6
+request-interval : 7
```

Comment:
It's funny how we keep having to increase the request interval with each new PR lol